static/javascripts/angular/route/controllers/create.route.controller.js

```diff
@@ -1,66 +1,140 @@
+            type: 'establishment',
+            country: $scope.details_city.route_details.address_components[country_index].short_name.toLowerCase()
           };
-          $scope.options_food = {
-               watchEnter: true,
-               types: 'establishment',
-               country: $scope.details_city.address_components[country_index].short_name.toLowerCase()
-          }
-          $scope.options_shop = {
-               watchEnter: true,
-               types: 'establishment',
-               country: $scope.details_city.address_components[country_index].short_name.toLowerCase()
-          }
         }
       });
-
-
       $scope.saveRoute = function() {
-
+          //console.log($scope.details_city);
+          //console.log($scope.details_food.food_details);
+          //console.log($scope.food_star_rating.star_rating);
+          //console.log($scope.food_budget.budget);
+          //console.log($scope.food_comment.comment);
             var places = [];
-            places.push({id: $scope.details_shop.place_id, type:0, location_lat:$scope.details_shop.geometry.location.lat(), location_lng:$scope.details_shop.geometry.location.lng()});
-            places.push({id: $scope.details_museum.place_id, type:1, location_lat:$scope.details_museum.geometry.location.lat(), location_lng:$scope.details_museum.geometry.location.lng()});
-            places.push({id: $scope.details_food.place_id, type:2, location_lat:$scope.details_food.geometry.location.lat(), location_lng:$scope.details_food.geometry.location.lng()});
+            places.push({
+              id: $scope.details_food.food_details.place_id,
+              type:0,
+              location_lat:$scope.details_food.food_details.geometry.location.lat(),
+              location_lng:$scope.details_food.food_details.geometry.location.lng(),
+              rating: $scope.food_star_rating.star_rating,
+              budget: $scope.food_budget.budget,
+              comment: $scope.food_comment.comment
+            });
+            //places.push({id: $scope.details_museum.place_id, type:1, location_lat:$scope.details_museum.geometry.location.lat(), location_lng:$scope.details_museum.geometry.location.lng()});
+            //places.push({id: $scope.details_food.place_id, type:2, location_lat:$scope.details_food.geometry.location.lat(), location_lng:$scope.details_food.geometry.location.lng()});
             var routeJSON = {
-                title: $scope.title,
-                description: $scope.description,
+                title: $scope.title.route_title,
+                description: $scope.description.route_description,
                 url_title: 'url_title',
                 places: places
             }
+            console.log(routeJSON);
             RouteService.createRoute(routeJSON).then(function(res) {
                 console.log("Res:"+ res.config.data.id);
                 console.log("Res:"+ res.config.data.title);
             })
       }
   }]);
-});
+});
```

Comment:
@cansintartici As a convention, let's follow a number of spaces for tab. It may be 4 spaces.